### PR TITLE
Added eurkey Swedish tap dance macro and private layout.

### PR DESCRIPTION
**Pull request by hvpkod**
## Description

Keymap to be used with Eurkey for easy access to Swedish characters on the #first layer.

Links:

- https://eurkey.steffen.bruentjen.eu/

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* NA

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+ *                   |      |      |      |/       /         \      \ |      |      |      |
+ *                   `----------------------------'           '------''--------------------'
+ */
+
+ [_QWERTY] = LAYOUT( \
+  KC_ESC,   KC_1,   KC_2,    KC_3,    KC_4,    KC_5,                     KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC, \
+  LT(_NAVI,KC_TAB),   KC_Q,   KC_W,    KC_E,    KC_R,    KC_T,                     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_MINS, \
+  KC_LSFT, KC_A,   KC_S,    KC_D,    KC_F,    KC_G,                     KC_H,    KC_J,    KC_K,    KC_L,     TD(TD1), TD(TD2), \
+  KC_LCTRL,  KC_Z,   KC_X,    KC_C,    KC_V,    KC_B, MT(MOD_LCTL,KC_LBRC),  MT(MOD_LALT,KC_RBRC),  KC_N,    KC_M,    KC_COMM, KC_DOT,  TD(TD3),  MT(MOD_RSFT,KC_ENT), \
+                             KC_LALT, KC_LGUI, LOWER,MT(MOD_LSFT,KC_SPC),   LT(_RAISE,KC_ENTER),   KC_ENTER,   KC_BSPC, KC_RGUI \
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+  return true;
+}
+
+// Tap dance function for enable swedish characters on first layer. Unregister to not let tap bleed over to next keypress. 
+// Tap dance 1
+void dance_1_finished(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    register_code(KC_SCLN);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code(KC_SCLN);
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_SCLN);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+  }
+  else
+  {
+    register_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code16(ALGR(KC_O));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+    register_code(KC_O);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_O);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+
+void dance_1_reset(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    unregister_code(KC_SCLN);
+  }
+  else
+  {
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    unregister_code16(ALGR(KC_O));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 9]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_O);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 10]
```diff
@@ -0,0 +1,357 @@
...
+  }
+}
+
+// Tap dance 2
+void dance_2_finished(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    register_code(KC_QUOT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code(KC_QUOT);
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 11]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_QUOT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 12]
```diff
@@ -0,0 +1,357 @@
...
+  }
+  else
+  {
+    register_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code16(ALGR(KC_A));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 13]
```diff
@@ -0,0 +1,357 @@
...
+    register_code(KC_A);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 14]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 15]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_A);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 16]
```diff
@@ -0,0 +1,357 @@
...
+
+void dance_2_reset(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    unregister_code(KC_QUOT);
+  }
+  else
+  {
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    unregister_code16(ALGR(KC_A));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 17]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_A);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 18]
```diff
@@ -0,0 +1,357 @@
...
+  }
+}
+
+// Tap dance 3
+void dance_3_finished(qk_tap_dance_state_t *state, void *user_data)
+{
+  // if (state->count == 2)
+  if (state->count == 2)
+  {
+    register_code(KC_SLSH);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code(KC_SLSH);
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 19]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_SLSH);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 20]
```diff
@@ -0,0 +1,357 @@
...
+  }
+  else
+  {
+    register_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    tap_code16(ALGR(KC_W));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 21]
```diff
@@ -0,0 +1,357 @@
...
+    register_code(KC_W);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 22]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 23]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_W);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 24]
```diff
@@ -0,0 +1,357 @@
...
+
+void dance_3_reset(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    unregister_code(KC_SLSH);
+  }
+  else
+  {
+    unregister_code(KC_RALT);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
    unregister_code16(ALGR(KC_W));
```

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 25]
```diff
@@ -0,0 +1,357 @@
...
+    unregister_code(KC_W);
```
Comment (fauxpark) [inline keyboards/lily58/keymaps/hvp/keymap.c]:
```suggestion
```

**Comment by fauxpark** [comment]
@hvpkod sorry I missed this one!

File: keyboards/lily58/keymaps/hvp/keymap.c [hunk 26]
```diff
@@ -0,0 +1,357 @@
...
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+
+extern keymap_config_t keymap_config;
```
Comment (drashna) [inline keyboards/lily58/keymaps/hvp/keymap.c]: This section isn't needed, and should be removed.

**Pull request closed without merging**